lib/modules/team_stats.rb

```diff
@@ -1,16 +1,26 @@
 module TeamStats
 
   def best_season(team_id)
+    win_pct_by_season(team_id)
```

Comment:
These methods are nice and short. I bet we have a lot of the same helper methods that can be condensed into fewer lines for our overall project.